storybookjs/storybook
# Components: reduce size of syntax highlighter

Issue: https://github.com/storybookjs/storybook/issues/9282

## What I did

It turns out that the solution was pretty simple. We were already importing the "light build" which doesn't include prebundled dependencies, but `react-syntax-highlighter` imports all languages anyway in their main package export. Importing from a specific file makes it work correctly. 

I added a comment in to that effect in the code, so that it can be removed later.

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes in the sense that they should not cause any visual changes.
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
